docs/001_develop/03_client-capabilities/023_custom-components/03_state-management.mdx

```diff
@@ -302,20 +321,22 @@ export class MyButton extends GenesisElement {
 
 Initial State:
 
-    The button displays "Clicked: 0 times".
+  > The button displays "Clicked: 0 times".
 
 After Clicks:
 
-    - First click: The button displays "Clicked: 1 times".
-    - Second click: The button displays "Clicked: 2 times".
-    - And so on, incrementing with each click.
-    - The button displays "Clicked: 0 times".
+  >- First click: The button displays "Clicked: 1 times".
+  >- And so on, incrementing with each click.
+  >- The button displays "Clicked: 0 times".
 
-Each time count updates, `countChanged` logs the old and new values to the console:
+Each time `count` updates, `countChanged` logs the old and new values to the console:
 
 ```typescript
 Count changed from 0 to 1
 Count changed from 1 to 2
 ```
+:::important
+
+`@observable` decorator is an effective way to manage component state, allowing properties to reactively update the UI, trigger callbacks, and handle complex data in a streamlined way.
```

Comment:
Maybe add an additional setence saying that properties reflected via an attribute using `@attr` are also observable and reactive in the same way